### PR TITLE
UI enhancements and SQL fixes

### PR DESCRIPTION
**Pull request by alondhe**
Fixes:
#65
#81
#79
#93

Enhancements:
#86
#80
#77

### CODE REVIEW

File: inst/sql/sql_server/concept_plausible_gender.sql [hunk 1]
```diff
@@ -13,18 +13,19 @@ plausibleGender = @plausibleGender
...
 	SELECT COUNT_BIG(*) AS num_violated_rows
 	FROM
 	(
-		SELECT @cdmTableName.* 
-		FROM @cdmDatabaseSchema.@cdmTableName
-			INNER JOIN @cdmDatabaseSchema.person
-			ON @cdmTableName.person_id = person.person_id
-		WHERE @cdmFieldName = @conceptId
-		AND person.gender_concept_id <> {@plausibleGender == 'Male'} ? {8507} : {8532} 
+		SELECT A.*
```
Comment (fdefalco) [inline inst/sql/sql_server/concept_plausible_gender.sql]: Would recommend aliases that were more descriptive than A/B.

File: inst/sql/sql_server/field_cdm_field.sql [hunk 2]
```diff
@@ -7,17 +7,21 @@ Parameters used in this template:
 cdmDatabaseSchema = @cdmDatabaseSchema
 cdmTableName = @cdmTableName
 cdmFieldName = @cdmFieldName
+isPostgresql = @isPostgresql
```
Comment (fdefalco) [inline inst/sql/sql_server/field_cdm_field.sql]: Is this not handled by SqlRender? Unclear as why there is DB specific instruction.  Might just need to understand.

File: inst/sql/sql_server/concept_plausible_gender.sql (hunk 1 above)
Comment (alondhe) [reply inst/sql/sql_server/concept_plausible_gender.sql]: Latest commit uses more descriptive aliases

File: inst/sql/sql_server/field_cdm_field.sql (hunk 2 above)
Comment (alondhe) [reply inst/sql/sql_server/field_cdm_field.sql]: Misunderstanding on my part -- it's not that PG was an issue, it was the casts added in the CSV. I've removed this hack.

**Comment by alondhe** [comment]
@fdefalco and @clairblacketer -- can you review now? I've updated the aliases to be more descriptive, removed the isPostgresql hack, and removed the cast as datetimes from the field level CSV and simply placed that into the relevant SQL templates.

**Comment by alondhe** [comment]
Thanks @fdefalco for the review!

**Pull request merged**